src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,20 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        Truck truck = new Truck();
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
+
+        Machine[] array = {truck, bulldozer, excavator};
+
+        for (Machine machine : array) {
+            machine.doWork();
+            machine.stopWork();
+            System.out.println("----------------");
```

Comment:
don't forget to remove all printlns before committing

Reply:
i think its more comfortable for eyes when user can see spaces or lines(like in thas case) to separate information from each other. But if it needed for the task its okay, i can do it

Reply:
You can use System.out.println for testing locally, but it should not be in the code